NewForce-Cohort-9/bad-blood-brotherhood
# Tyler branch3

## What changed?

Complete overhaul of import/exports for Greenland and Cambodia for modularity.  Overhaul of HTML on both to be compatible.  Imports/exports for Lichtenshtein (lol) nearly done, have not updated HTML yet.

## How to test?

Hop on over to my branch and check it out.  Run it on the site to verify that you can see the information that's being imported in the Greenland and Cambodia sections (you will not see anything for Stein yet.)
